sandbox/src/Http/RequestBuilder.php

```diff
@@ -0,0 +1,217 @@
+<?php
+namespace YotiSandbox\Http;
+
+use Yoti\Entity\Profile;
+use YotiSandbox\Entity\SandboxAgeVerification;
+use YotiSandbox\Entity\SandboxAnchor;
+use YotiSandbox\Entity\SandboxAttribute;
+
+class RequestBuilder
+{
+    /**
+     * @var string
+     */
+    private $rememberMeId;
+
+    /**
+     * @var array
+     */
+    private $sandboxAttributes = [];
+
+    /**
+     * @param string $value
+     */
+    public function setRememberMeId($value)
+    {
+        $this->rememberMeId = $value;
+    }
+
+    public function setFullName($value, $optional = 'false', array $anchors = [])
+    {
+        $this->addAttribute($this->createAttribute(
+            Profile::ATTR_FULL_NAME,
+            $value,
+            '',
```

Comment:
Since PHP doesn't have named parameters, do you think it would be useful to set `$noDerivation = ''`, and then use `$noDerivation` (or similar) in each of these calls to createAttribute? I think it would be helpful when looking at the code, rather than having to navigate to `createAttribute` to see what this argument is